mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+/* Math.random function returns a pseudo random value between 0 to 1 and multiplies it by 10.*/
```

Comment:
Is the range inclusive of 1? And consequently, will the range of output from your method be inclusive of 10?

Reply:
Added a bit more detail to the comment, Thanks Karen.